doc/classes/Image.xml

```diff
@@ -481,6 +485,7 @@
 			<description>
 				Saves the image as a JPEG file to [param path] with the specified [param quality] between [code]0.01[/code] and [code]1.0[/code] (inclusive). Higher [param quality] values result in better-looking output at the cost of larger file sizes. Recommended [param quality] values are between [code]0.75[/code] and [code]0.90[/code]. Even at quality [code]1.00[/code], JPEG compression remains lossy.
 				[b]Note:[/b] JPEG does not save an alpha channel. If the [Image] contains an alpha channel, the image will still be saved, but the resulting JPEG file won't contain the alpha channel.
+				[b]Note:[/b] This method is only available in engine builds with the JPG module enabled. By default, the JPG module is enabled, but it can be disabled at build-time using the [code]module_jpg_enabled=no[/code] SCons option.
```

Comment:
```suggestion
				[b]Note:[/b] This method is only available in engine builds with the JPG module enabled. By default, the JPEG module is enabled, but it can be disabled at build-time using the [code]module_jpg_enabled=no[/code] SCons option.
```